modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/expression/MBDecision.java

```diff
@@ -133,8 +301,28 @@ public Expression decisionAll(){
      *          [""any"", boolean, boolean, ...]: boolean
      * @return
      */
-    public Expression decisionAny(){
-        return null;
+    private Expression decisionAny() {
+        // validate the arg list
+        if (json.size() < 2) {
+            throwInsufficientArgumentCount(ALL, 1);
+        }
+        // first arg needs to be the operator
+        final String anyOp = parse.get(json, 0);
+        if (!ANY.equals(anyOp)) {
+            // not a ANY(any) op
+            throwUnexpectedOperationException(ANY, anyOp);
+        }
+        // loop over each expression and evaluate. If an expression evaluates to true, we are done.
+        for (int i = 1; i < json.size(); ++i) {
+            Expression expression = parse.string(json, i);
+            // if the expression evaluates to true, we are done
+            final Boolean evaluate = expression.evaluate(expression, Boolean.class);
```

Comment:
Expressions cannot be evaluated at parse time, in case they contain references to properties of the data that are only available at render time (e.g. attributes)